plugins/woocommerce/tests/e2e-pw/tests/admin-tasks/payment.spec.js

```diff
@@ -8,7 +8,7 @@ test.describe( 'Payment setup task', () => {
 	test.beforeEach( async ( { page, baseURL } ) => {
 		// Skip skipping the setup wizard if the core profiler is enabled.
 		// When the core-profiler is enabled, the following code won't work, causing the tests to fail.
-		if ( ! features.is_enabled( 'core-profiler' ) ) {
+		if ( ! features.isEnabled( 'core-profiler' ) ) {
```

Comment:
I renamed all the features helpers to camel case for consistency.

Reply:
This has been addressed in https://github.com/woocommerce/woocommerce/pull/40260. Thank you for bringing it up 👍